packages/runtimes/js/src/helpers/bundle-manifest.js

```diff
@@ -1,18 +1,17 @@
-var mapping = {};
+var mapping = new Map();
```

Comment:
Are we ok to drop browsers that don't support `Map`? I think Parcel's output still worked in IE11 up until this point. Maybe it's ok, but wonder if it should be done in a major version?

Reply:
I believe Map is supported in IE11. This support coverage looks good enough to me. But I can remove if you want.
https://caniuse.com/mdn-javascript_builtins_map